UPGRADING.rst

```diff
@@ -10,6 +10,32 @@ branch that does not have the listed changes, the steps would need to be
 reverted. This is all fairly informal and loosely defined. Hopefully we won't
 have too many entries in this file.
 
+#3497 Fix: API Gateway log is missing important fields
+======================================================
+
+Upgrades to the logs format for API Gateway require recreating
```

Comment:
Too complicated. There must a simpler way to get the provisioner to run.

https://www.terraform.io/cli/commands/taint